docs/source/changelog.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 *********
 
+Current version
+===============
+
+Changes
+-------
+
+* Remove support for Python 3.6; add support for up to Python 3.11.
```

Comment:
May I suggest following NEP29 and dropping py3.7 too? It would make it easier for folks to know what python versions you're likely to support at the point of release.

Reply:
Thanks; see 74b84252340d27592f68b82e5a1793be311d19b2